app/experimenter/experiments/tests/test_serializers.py

```diff
@@ -877,6 +877,34 @@ def test_serializer_rejects_duplicate_branch_names(self):
         self.assertFalse(serializer.is_valid())
         self.assertIn("variants", serializer.errors)
 
+    def test_serializer_outputs_dummy_variants_when_no_variants(self):
+        experiment = ExperimentFactory.create(type=ExperimentConstants.TYPE_GENERIC)
+
+        serializer = ExperimentDesignBaseSerializer(experiment)
+
+        self.assertCountEqual(
+            serializer.data,
+            {
+                "type": ExperimentConstants.TYPE_GENERIC,
+                "variants": [
+                    {
+                        "description": None,
+                        "is_control": True,
+                        "name": None,
+                        "ratio": None,
+                        "value": None,
+                    },
+                    {
+                        "description": None,
+                        "is_control": False,
+                        "name": None,
+                        "ratio": None,
+                        "value": None,
```

Comment:
oh yikes, this test shouldn't be passing if value is present.... seems like I've done something weirder than inputting the wrong types 👀

Reply:
I think you might need to break this test and compare one field/value at a time. Apparently i misused CountEqual somehow here...
<img width="1399" alt="Screen Shot 2019-11-05 at 7 47 23 PM" src="https://user-images.githubusercontent.com/25379936/68267053-dbc24580-0005-11ea-89b1-2f34153c2819.png">